### DISCUSSION TITLE
官方文档里面组件能否按字母顺序排序，这样查找会更快

### DISCUSSION THREAD

**Discussion by liyefox** [ant-design/ant-design#36313] [General]
### What problem does this feature solve?

现在查找组件，知道大概的组件名了，但是滚动半天也不知道组件在何处？能否加个按字母排序的功能，现在只能按command k来搜

### What does the proposed API look like?

加个按字母排序的功能，现在只能按command k来搜

**Comment by afc163** [comment]
直接按字母排序的话，组件分类不好处理，另外现在单个组件分类下已经是按字母顺序排列的了。
